templates/speaker-template.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * The template for displaying the single speaker posts
+ *
+ * @since   1.0.0
+ * @package wp_conference_schedule_pro
+ */
+
+get_header(); ?>
+
+	<section id="primary" class="content-area">
+		<main id="main" class="site-main">
+
+			<?php
+			/* Start the Loop */
+			while ( have_posts() ) : the_post();
+				$post_id           = get_the_ID();
+				$first_name        = get_post_meta( $post_id, 'wpcsp_first_name', true );
+				$last_name         = get_post_meta( $post_id, 'wpcsp_last_name', true );
+				$full_name         = $first_name . ' ' . $last_name;
+				$title             = get_post_meta( $post_id, 'wpcsp_title', true );
+				$organization      = get_post_meta( $post_id, 'wpcsp_organization', true );
+				$schedule_page_url = get_option( 'wpcs_field_schedule_page_url' );
+				$speaker_page_url  = get_option( 'wpcsp_field_speakers_page_url' );
+
+				function wpcsp_get_social_links() {
+					$social_icons = [];
+					foreach ( [ 'facebook', 'twitter', 'instagram', 'linkedin', 'youtube', 'website' ] as $social_icon ) {
+						$url = get_post_meta( get_the_ID(), 'wpcsp_' . $social_icon . '_url', true );
+						if ( $url ) {
+
+							$social_label = $social_icon;
+							if ( $social_icon == 'website' ) {
+								$social_icon = 'admin-site-alt3';
+							}
+							if ( $social_icon == 'facebook' ) {
+								$social_icon = 'facebook-alt';
+							}
+
+							ob_start();
+
+							echo '<a class="wpcsp-speaker-social-icon-link" href="' . esc_url( $url ) . '" target="_blank" aria-label="' . $social_label . '"><span class="dashicons dashicons-' . $social_icon . '"></span></a>';
+
+							$social_icons[] = ob_get_clean();
```

Comment:
No need to use `ob_` methods here, just save the value into the variable instead of `echo`.